src/cameraGeom/Detector.cc

```diff
@@ -141,9 +153,12 @@ void Detector::_init() {
 //
 // Explicit instantiations
 //
-#define INSTANTIATE(FROMSYS, TOSYS)                                                                 \
-    template std::shared_ptr<geom::TransformPoint2ToPoint2> Detector::getTransform(FROMSYS const &, \
-                                                                                   TOSYS const &) const;
+#define INSTANTIATE(FROMSYS, TOSYS)                                                                          \
+    template std::shared_ptr<geom::TransformPoint2ToPoint2> Detector::getTransform(FROMSYS const &,          \
+                                                                                   TOSYS const &) const;     \
+    template geom::Point2D Detector::transform(geom::Point2D const &, FROMSYS const &, TOSYS const &) const; \
+    template std::vector<geom::Point2D> Detector::transform(std::vector<geom::Point2D> const &,              \
```

Comment:
For the same reason we don't line up equal signs in a list of equations, I suggest not spacing out the backslashes.

Reply:
clang-format